src/PhpSpreadsheet/Calculation/Statistical.php

```diff
@@ -3,6 +3,8 @@
 namespace PhpOffice\PhpSpreadsheet\Calculation;
 
 /* LOG_GAMMA_X_MAX_VALUE */
+use PhpOffice\PhpSpreadsheet\Calculation;
+
 define('LOG_GAMMA_X_MAX_VALUE', 2.55e305);
```

Comment:
There are indeed a few constants that should be refactored into class constants. But that should be a separate work. I just created #157 for that.